testing/test_plan.md

```diff
@@ -0,0 +1,135 @@
+# Test plan
+
+## Environment
+
+-   OS: XXX (Windows, macOS, latest Ubuntu LTS)
+-   Python
+    -   Distribution: XXX (CPython, miniconda)
+    -   Version: XXX (3.9, 3.10, 3.11)
+-   VS Code: XXX (Insiders)
+-   Python && Pylance Extension: XXX (Insiders)
+
+## Tests
+
+### P0 Test Scenarios
+
+#### Basic language services
+
+-   [ ] Errors
+    1. Open a new python file
+    1. Type invalid python into it. Like `x2+2-3` on a single line.
+    1. Verify you get a red squiggle
+    1. Type valid python after that. 
+    1. Make sure no squiggles happen
+-   [ ] Import errors
+    1. Create a brand new python environment
+    1. Select this in VS code.
+    1. Open a new python file
+    1. Add `import pandas as pd` at the top
+    1. It should error on the pandas
+    1. Open a terminal
+    1. Activate your new environment

```

Comment:
I think the environment is automatically activated in the terminal? Might be a recent enhancement?

Reply:
Doesn't always seem to work for me.